### PR TITLE
New plugin: somfytahoma

### PR DESCRIPTION
**Pull request by fetzerch**
Add support for controlling Somfy smart home devices through their
Tahoma box (and the Somfy API).

The plugin currently contains support for roller shutters as well as
for venetian blinds.

---

nymea-plugins pull request checklist:

- [x] Make sure the pull request's title is of format "Plugin name: Add support for xyz" or "New plugin: Plugin name"

- [x] Did you test the changes on hardware, if not (e.g. absence of required hardware), please mention a person to confirm it has been tested.

- [x] Did you update the plugin's README.md accordingly?

- [x] Did you update translations (`cd builddir && make lupdate`)?

- [x] If you added a new plugin, should it be added to nymea-plugins-all or ~nymea-plugins-maker~?

### CODE REVIEW

**Review by mzanetti — CHANGES_REQUESTED** [review]
Looks quite good. There's an issue with the meta.json / logo.
It refers to somfy.svg but there's a somfy.png.

Ideally you'd provide a svg

File: somfytahoma/somfytahoma.pro [hunk 1]
```diff
@@ -0,0 +1,11 @@
+include($$[QT_INSTALL_PREFIX]/include/nymea/plugin.pri)
```
Comment (mzanetti) [inline somfytahoma/somfytahoma.pro:1]: can you include ../plugin.pri instead please in here

Comment (fetzerch) [reply somfytahoma/somfytahoma.pro]: sure, done

**Comment by fetzerch** [comment]
I've updated the change to include the logo as svg instead of png.

**Review by mzanetti — APPROVED** [review]
Ack. Thank you very much.

Will be moved into the testing repository now

**Pull request merged**